ivy/functional/backends/paddle/activations.py

```diff
@@ -22,10 +22,18 @@
 
 
 @with_supported_dtypes(
-    {"2.6.0 and below": ("float32", "float64", "complex")},
+    {
+        "2.6.0 and below": (
+            "float32",
+            "float64",
+            "complex64",
+        )
+    },
     backend_version,
 )
-def relu(x: paddle.Tensor, /, *, out: Optional[paddle.Tensor] = None) -> paddle.Tensor:
+def relu(
+    x: paddle.Tensor, /, *, complex_mode="jax", out: Optional[paddle.Tensor] = None
+) -> paddle.Tensor:
     if paddle.is_complex(x):
```

Comment:
```python
import ivy
import paddle
import jax
x = paddle.to_tensor([1-1j, 1+1j])
y = jax.numpy.array([1-1j, 1+1j])
#paddle output: ivy.array([1.+0.j, 1.+1.j])
print(ivy.relu(x))
#jax output: ivy.array([1.-1.j, 1.+1.j])
print(ivy.relu(y))
```
as the logic for the complex dtype at  paddle is `return paddle.complex(F.relu(x.real()), F.relu(x.imag()))` it's dosen't pass this case bcz the x.imag() part is -ve and if we apply the `relu` activation function then it returns the `zero` that is the wrong logic here for paddle backend for complex dtype.